src/main/java/nl/tudelft/ti2206/group9/gui/ExternalTicker.java

```diff
@@ -214,5 +228,59 @@ private static PhongMaterial chooseMaterial() {
 			return Style.MOSS;
 		}
 	}
+
+	/**
+	 * Render the countdownLabel method.
+	 * @param index The index to be rendered.
+	 */
+	public void countdown(final Integer index) {
+		final int textSize = 64;
+		final int offsetX = 20;
+		final int offsetY = 50;
+
+		countdownLabel.setText(index.toString());
+		countdownLabel.setFont(Font.font("Roboto", FontWeight.BOLD, textSize));
+		countdownLabel.setTextFill(Color.WHITE);
+		countdownLabel.setLayoutX(ShaftEscape.WIDTH / 2 - offsetX);
+		countdownLabel.setLayoutY(ShaftEscape.HEIGHT / 2 - offsetY);
+
+		countdownAnimation(index);
+	}
+
+	/**
+	 * Render the animations for the countdownLabel method.
+	 * @param index The index to be rendered.
+	 */
+	private void countdownAnimation(
+			final Integer index) {
+		final int duration = 400;
+
+		final ScaleTransition st = new ScaleTransition(
+				Duration.millis(duration), countdownLabel);
+		st.setByY(1);
+		st.setByX(1);
+		st.setCycleCount(2);
+		st.setAutoReverse(true);
+		st.play();
+
+		final FadeTransition ft = new FadeTransition(
+				Duration.millis(duration), countdownLabel);
+		ft.setAutoReverse(true);
+		ft.setCycleCount(2);
+		ft.setToValue(1);
+		ft.setFromValue(0);
+		ft.play();
+
+		st.setOnFinished(new EventHandler<ActionEvent>() {
+			public void handle(final ActionEvent event) {
+				if (index > INT_ONE) {
```

Comment:
I think I get what you mean. I hope that you're able to refactor this to something without the `INT_ONE` constant. Maybe `if (index - 1 > 0)`?

<review, logic, configuration